pkg/kubelet/kuberuntime/convert.go

```diff
@@ -30,6 +30,7 @@ func toKubeContainerImageSpec(image *runtimeapi.Image) kubecontainer.ImageSpec {
 
 	if image.Spec != nil && len(image.Spec.Annotations) > 0 {
 		annotationKeys := make([]string, 0, len(image.Spec.Annotations))
+		annotations = make([]kubecontainer.Annotation, 0, len(image.Spec.Annotations))
 		for k := range image.Spec.Annotations {
 			annotationKeys = append(annotationKeys, k)
 		}
```

Comment:
the only purpose of the second loop as far as I can see is to have sorted annotations. I'm just wondering if we really need them sorted? If we don't, then we can improve performance of this function even more by filling `annotations` in a single loop.

Reply:
By looking for the upstream method that calls toKubeContainerImageSpec, no code that needs to sort Annotations is found. Therefore, I removed the sorting logic. @bart0sh PTAL, thanks.

Reply:
Thanks. Looks good to me, but I'd like other people to look at it.